smartcards/cli.py

```diff
@@ -0,0 +1,50 @@
+import json, hashlib, time, sys
+import click
+from flask.cli import AppGroup
+
+from smartcards.card import CardInterface
+
+card_cli = AppGroup('card')
+
+@card_cli.command('read')
+def read():
+    time.sleep(2)
+
+    short_value, has_long_value = CardInterface.read()
+    if not short_value:
+        print('error: unable to read short_value')
+        sys.exit(-1)
+
+    print(short_value.decode('utf-8'))
+
+    if has_long_value:
+        print("reading long value...")
+        long_value = CardInterface.read_long()
+        print(long_value.decode('utf-8'))
+
+    print("done")
+
+@card_cli.command('write')
+@click.option('-t', '--type', default='voter', type=click.Choice(['voter', 'admin', 'clerk']))
+@click.option('-f', '--file', type=click.File('r'))
```

Comment:
Allow input via stdin if this is `-`, so commands can be chained together. Magic 🎉